components/application-mgt/org.wso2.carbon.identity.application.mgt/src/test/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImplTest.java

```diff
@@ -632,48 +644,84 @@ public void testCreateAndGetApplicationWithProtocolService() throws IdentityAppl
         InboundProtocolsDTO inbounds = setInboundProtocol();
         inbounds.addProtocolConfiguration(() -> ApplicationConstants.StandardInboundProtocols.SAML2);
         applicationDTOBuilder.inboundProtocolConfigurationDto(inbounds);
-        
-        // Mocking protocol service.
-        ApplicationManagementServiceComponentHolder.getInstance().addApplicationInboundAuthConfigHandler(
-                customSAML2InboundAuthConfigHandler());
-        
-        // Creating application.
-        applicationManagementService.createApplication(applicationDTOBuilder.build(), SUPER_TENANT_DOMAIN_NAME,
-                USERNAME_1);
+
+        try (MockedStatic<OrganizationManagementUtil> organizationMgtUtilMockedStatic =
+                     mockStatic(OrganizationManagementUtil.class)) {
+            organizationMgtUtilMockedStatic.when(() -> OrganizationManagementUtil.isOrganization(
+                    SUPER_TENANT_DOMAIN_NAME)).thenReturn(isOrganization);
```

Comment:
```suggestion
                   ArgumentMatchers.anyString()).thenReturn(isOrganization);
```
check whether this is possible

Reply:
anyString() is used in other places as well. We can do this. but since the application creation and deletion is done based on the tenant name, isn't it better to use the same for this mock as well?

Reply:
Updated